certified-connectors/Integration Toolbox/apiDefinition.swagger.json

```diff
@@ -6,7 +6,7 @@
     "description": "The Integration Toolbox provides a set of operations which transform data in various ways.",
     "contact": {
       "name": "Integration Toolbox support",
-      "url": "https://www.integration-toolbox.com",
+      "url": "https://integration-toolbox.com",
```

Comment:
@lfalck Both are same and we expect the url values are fully qualified website URLs. Can you please revert back the changes of url. If you have any specific reason, please let us know.

Reply:
We are just removing the www subdomain from the link, it is still a fully qualified url.